### PR TITLE
Use CLI Helpers for output formatting

### PR DESCRIPTION
**Pull request by tsroten**
## Description

This pull request moves all output formatting to CLI Helpers. It fixes https://github.com/dbcli/pgcli/issues/617.

It removes:
- the packaged tabulate library.
- the expanded output module.

One other change is that the expanded output starts the record numbers at 1 instead of 0, which actually makes it behave like psql. So, instead of:
`-[ RECORD 0 ]-------------------------`
you see:
`-[ RECORD 1 ]-------------------------`

~~**Note: The tests will fail until CLI Helpers v2.0 is released.**~~

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

File: pgcli/main.py [hunk 1]
```diff
@@ -156,6 +157,7 @@ def __init__(self, force_passwd_prompt=False, never_passwd_prompt=False,
         self.now = dt.datetime.today()
 
         self.completion_refresher = CompletionRefresher()
+        self.formatter = TabularOutputFormatter()
```
Comment (amjith) [inline pgcli/main.py]: I'm mildly curious. Is there a reason for this to be an instance variable? 

Why not instantiate the TabularOutputFormatter() inside the format_output() and use it?

Comment (tsroten) [inline pgcli/main.py]: Not really, I'll change it 😄

Comment (tsroten) [inline pgcli/main.py]: Ok, done. 👍

File: pgcli/main.py [hunk 2]
```diff
@@ -757,6 +759,48 @@ def get_last_query(self):
         """Get the last query executed or None."""
         return self.query_history[-1][0] if self.query_history else None
 
+    def format_output(self, title, cur, headers, status, settings):
```
Comment (amjith) [inline pgcli/main.py]: With the new change it looks like format_output doesn't need to be part of the PGCli class. The self variable is not used anywhere. 

We can either make it a @staticmethod if we think that's appropriate or we can move it outside of this class. What do you think?

Comment (tsroten) [inline pgcli/main.py]: Good point. I'll move it out of the class and update the tests.

Comment (tsroten) [inline pgcli/main.py]: Done 😄

**Comment by amjith** [comment]
This looks great!! Fantastic work. 👏  🍰

**Pull request merged**